lib/integrations/jira.rb

```diff
@@ -0,0 +1,84 @@
+require "integrations/base"
+
+module Integrations
+  class Jira < Base
+    def self.key
+      'jira'
+    end
+
+    def send_event
+      return false unless has_failed_tests?
```

Comment:
Nope, it supports only those with specific test(s), I don't see how it would be useful otherwise as they are not bugs?